ALEC-227: ALEC docs: update installation section (#101)
OpenNMS/alec: docs/modules/admin/pages/upgrading.adoc

:imagesdir: ../assets/images
= Upgrade ALEC

This section describes how to upgrade ALEC on OpenNMS and Sentinel.

== Upgrade ALEC on OpenNMS

> NOTE: The following instructions apply to both basic and distributed ALEC deployments.

Reviewer: ```suggestion
NOTE: The following instructions apply to both basic and distributed ALEC installations.
```

> = Upgrade ALEC
> :imagesdir: ../assets/images

Reviewer: ```suggestion

:imagesdir: ../assets/images
= Upgrade ALEC
```

> This section includes instructions to upgrade ALEC on OpenNMS and Sentinel.

Reviewer: ```suggestion
This section describes how to upgrade ALEC on OpenNMS and Sentinel.
```

> == Upgrade ALEC on OpenNMS (basic and distributed deployments)

Reviewer: ```suggestion
== Upgrade ALEC on OpenNMS

NOTE: The following instructions apply to both basic and distributed ALEC deployments.
```

> Stop OpenNMS:

Reviewer: I'd change all of these to code blocks as well, just for consistency.

sudo /etc/init.d/opennms restart
```

> == Upgrade ALEC on Sentinel (distributed deployment)

Reviewer: ```suggestion
== Upgrade ALEC on Sentinel

NOTE: The following instructions apply to only a distributed ALEC installation.
```

> Here we assume that ALEC is already up and running on one or more Sentinel instances.
> For each Sentinel instance, do the following:

Reviewer: ```suggestion
For this procedure, we assume that ALEC is already installed and running on one or more Sentinel instances in your network.
Follow the steps below for each Sentinel instance:
```

> Stop Sentinel:

Reviewer: Same as previous code block suggestions.
